docs/run-node/run-node.md

```diff
@@ -0,0 +1,90 @@
+<!--
+order: 2
+-->
+
+# Running a Node
+
+Now that the application is ready and the keyring populated, it's time to see how to run the blockchain node. In this section, the application we are running is called [`simapp`](https://github.com/cosmos/cosmos-sdk/tree/v0.40.0-rc1/simapp), and its corresponding CLI binary `simd`. {synopsis}
+
+## Pre-requisite Readings
+
+- [Anatomy of an SDK Application](../basics/app-anatomy.md) {prereq}
+- [Setting up the keyring](./keyring.md) {prereq}
+
+## Run the Chain
+
+::: warning
+Make sure you can build your own binary, and replace `simd` with the name of your binary in the snippets.
+:::
+
+Before actually running the node, we need to initialize the chain, and most importantly its genesis file. This is done with the `init` subcommand:
+
+```bash
+# The argument <moniker> is the custom username of your node, it should be human-readable.
+simd init <moniker> --chain-id my-test-chain
+```
+
+The command above creates all the configuration files needed for your node to run, as well as a default genesis file, which defines the initial state of the network. All these configuration files are in `~/.simapp` by default, but you can overwrite the location of this folder by passing the `--home` flag.
+
+The `~/.simapp` folder has the following structure:
+
+```bash
+.                                   # ~/.simapp
+  |- data                           # Contains the databases used by the node.
+  |- config/
+      |- app.toml                   # Application-related configuration file.
+      |- config.toml                # Tendermint-related configuration file.
+      |- genesis.json               # The genesis file.
+      |- node_key.json              # Tendermint node pubkey that will be used for signing blocks.
+      |- priv_validator_key.json    # TODO
+```
+
+Before starting the chain, you need to populate the state with at least one account. To do so, first [create a new account in the keyring](./keyring.md#adding-keys-to-the-keyring) named `my_validator` (feel free to choose another name).
+
+Now that you have created a local account, go ahead and grant it some `stake` tokens in your chain's genesis file. Doing so will also make sure your chain is aware of this account's existence:
+
+```bash
+simd add-genesis-account $(simd keys show my_validator -a) 100000000stake --chain-id my-test-chain
+```
+
+Note that the tokens in the SDK have the `{amount}{denom}` format.
```

Comment:
Please explain denom. I was initially confused, because I was usually used in context of value denomination. Example: 1USD denominated in US Cent is 100 USCent. Or 1 BTC is denominated in satoshi, where 1 BTC = 1e9 satoshi.

Reply:
We should also explain how "stake" just happens to be the denom in simapp, and for your own chain with its own staking denom, that should be used instead if following this guide.